vms/spdagvm/vm_test.go

```diff
@@ -803,7 +803,7 @@ func TestTxOutputOverflow(t *testing.T) {
 	// Send [2*txFeeTest+1] NanoAva from [key1Str] to [testAddr]
 	// Should overflow [testAddr] by 1
 	_, err = vm.Send(2*txFeeTest+1, "", testAddr, []string{privKey1})
-	if err == errOutputOverflow {
+	if err != errOutputOverflow {
```

Comment:
This test is definitely wrong as is, but I think the whole test should just be removed, I'm not sure what it was meant to be testing.

@danlaine thoughts?